services/core-web/src/components/common/config.ts

```diff
@@ -0,0 +1,36 @@
+import RenderAutoComplete from "@mds/common/components/forms/RenderAutoComplete";
+import RenderAutoSizeField from "@mds/common/components/forms/RenderAutoSizeField";
+import RenderCheckbox from "@mds/common/components/forms/RenderCheckbox";
+import RenderDate from "@mds/common/components/forms/RenderDate";
+import RenderField from "@mds/common/components/forms/RenderField";
+import RenderFileUpload from "@mds/common/components/forms/RenderFileUpload";
+import RenderGroupCheckbox from "@mds/common/components/forms/RenderGroupCheckbox";
+import RenderMultiSelect from "@mds/common/components/forms/RenderMultiSelect";
+import RenderRadioButtons from "@mds/common/components/forms/RenderRadioButtons";
+import RenderSelect from "@mds/common/components/forms/RenderSelect";
+import RenderCascader from "@mds/common/components/forms/RenderCascader";
+import RenderGroupedSelect from "@mds/common/components/forms/RenderGroupedSelect";
+import RenderLargeSelect from "@mds/common/components/forms/RenderLargeSelect";
+import RenderTime from "@mds/common/components/forms/RenderTime";
+import RenderLabel from "./RenderLabel";
+import RenderYear from "@mds/common/components/forms/RenderYear";
+
+export const renderConfig = {
```

Comment:
I deleted most of the CORE/MS input components, and just repointed this file (and the equivalent MS file) to the components in common.